javascriptv3/example_code/bedrock-runtime/README.md

```diff
@@ -0,0 +1,94 @@
+# Amazon Bedrock Runtime code examples for the SDK for JavaScript (v3)
+
+## Overview
+
+Shows how to use the AWS SDK for JavaScript (v3) to work with Amazon Bedrock Runtime.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+_Amazon Bedrock Runtime is a fully managed service that makes it easy to use foundation models from third-party providers and Amazon._
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account. For more details, see [AWS Pricing](https://aws.amazon.com/pricing/) and [Free Tier](https://aws.amazon.com/free/).
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `javascriptv3` folder.
+
+
+<!--custom.prerequisites.start-->
+> ⚠ You must request access to a model before you can use it. If you try to use the model (with the API or console) before you have requested access to it, you will receive an error message. For more information, see [Model access](https://docs.aws.amazon.com/bedrock/latest/userguide/model-access.html).
+<!--custom.prerequisites.end-->
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+- [Text generation with AI21 Labs Jurassic-2;](javascriptv3/example_code/bedrock-runtime/action/invoke-jurassic2.js) (`InvokeModel`)
```

Comment:
Extraneous semi-colon. Was this auto-generated?

Reply:
Good catch. This was a typo in the metadata and affects all languages. I've created a separate PR (#5946) that includes the fix and the newly generated readmes.